concent_api/core/message_handlers.py

```diff
@@ -331,7 +336,11 @@ def handle_send_force_subtask_results(request, client_message: message.concents.
             reason = message.concents.ServiceRefused.REASON.DuplicateRequest,
         )
 
-    if not base.is_provider_account_status_positive(request):
+    if not base.is_account_status_positive(  # pylint: disable=no-value-for-parameter
+        concent_rpc             = base.get_concent_rpc_instance(),  # pylint: disable=no-value-for-parameter
```

Comment:
Please hide this part in backends API. It is specific only for sci_backend. So:
1. Remove `concent_rpc` parameter from all base.py functions.
2. Remove `concent_rpc` parameter from all mock.py functions.
3. In `sci_backend`, call this function when you need to use RPC.
4. Remove all `get_concent_rpc_instance` functions.

Please fix for all usages and all backend functions.

Reply:
Done